Algorithm-Arguments-Optional.md

```diff
@@ -1,6 +1,8 @@
+# Algorithm Arguments Optional
+
 ![](http://i.imgur.com/63Iaec6.jpg)
 
-# Explanation:
+## Explanation:

```

Comment:
The Explanation should appear in chat at the first call. With `h2`, it won't appear.